File: MyApplication/app/src/main/java/com/example/myapplication/MainActivity.java
package com.example.myapplication;

import androidx.appcompat.app.AppCompatActivity;

import android.os.Bundle;
import android.view.View;
import android.widget.Toast;

public class MainActivity extends AppCompatActivity {
    
    int clickCount = 0;

    /** mainActivity 최초실행시 실행
    */
     @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        //UI 세팅
        setContentView(R.layout.activity_main);
        //Toast 메세지
        Toast.makeText(getApplicationContext(), "프로그래밍 시작", Toast.LENGTH_LONG).show();

        findViewById(R.id.button).setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                clickCount = clickCount +  1;

                String text = "";

                for(int i=0; i<clickCount;i++){
                    if(i>= 3){
                        break;
                    }
                    text = text + "HIHI";
                }
                Toast.makeText(getApplicationContext(), text, Toast.LENGTH_SHORT).show();
            }
        });

    }
}

File: MyApplication3/app/src/main/java/com/example/myapplication/MainActivity.kt
package com.example.myapplication

import android.media.MediaPlayer
import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.view.View
import android.widget.TextView

class MainActivity : AppCompatActivity() {

    lateinit var tv: TextView;
    lateinit var mp: MediaPlayer;

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        mp = MediaPlayer.create(this, R.raw.sample)
        mp.isLooping=false


        tv = this.findViewById(R.id.hellobutton)
        tv.setOnClickListener(View.OnClickListener {

            if(!mp.isPlaying){
                mp.start()
                tv.setText("Stop")
            }else{
                mp.pause()
                tv.setText("Start")
            }


        })

    }
}